assemble/src/main/scripts/create-jshell.sh

```diff
@@ -36,6 +36,25 @@ function addAccumuloAPI() {
   echo
 }
 
+function addClientBuild() {
+  echo "URL clientPropUrl = 
```

Comment:
If you want to avoid escaping double-quotes in the string, you can use a single quote to quote the echo parameter.

Even better, you can avoid the use of quotes entirely in bash, if you use a [heredoc](https://tldp.org/LDP/abs/html/here-docs.html). This is commonly used with `cat`, since multi-line strings don't always work well for `echo`:

```bash
cat <<EOF
This is a multi-line
string that does not need to be quoted,
#the pound sign does not get interpreted as a comment,
    spacing is treated literally,
and quotes (" or ') don't need to be escaped.
However, it does get treated like a double-quoted string
for the purposes of interpolation or embedding variables
like $HOME or ${USER^^} or expressions
like $(id -un) or $(echo "abzc" | tr -d 'z').
Try it by copying this block into your terminal.
EOF
```

The only annoying thing is the terminator for the multi-line string needs to start at the very beginning of the line (no indentation is allowed, whatsoever, even if you are inside a function or if-else block, or it won't work), and any indentation in the block is indented in the output, so although it's easier to read and write, it can throw off your script's style a little bit. But, it's probably worth it.